lib/useName.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react'
+import { getToken } from './userAuth'
+export const USER_NAME_URL = `${process.env.NEXT_PUBLIC_API_URL}/user`
+
+export default function useName( UserId, scores ) {
```

Comment:
By convention, Uppercase variables are treated as React Components (see note from [here](https://reactjs.org/docs/components-and-props.html#rendering-a-component)). In our case, there is no difference, but it's good to be respect the standards.

Please rename `UserId` to `userId`.

PS: I prefer not to insert spaces between variables and round paranthesis.